### PR TITLE
Backend database

### PR DESCRIPTION
**Pull request by valentinbreiz**

### CODE REVIEW

**Review by AdrienRoco — DISMISSED** [review]

File: routes/user.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  })
+  .catch((error) => {
+    // error changing email
+    res.status(400).send(`Error creating user: ${error}`);
+  });
+});
+
+// Get User Rights
+
+userRouter.get('/userRights', async (req, res) => {
```
Comment (valentinbreiz) [inline routes/user.js:135]: Why REST? We moved to GRPC.

File: routes/user.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+const firebase = initializeApp(firebaseConfig);
+const auth = getAuth(firebase);
+const database = getDatabase(firebase);
+const dbRef = ref(database);
+
+
+let user = null;
+
+// Login route
+userRouter.post('/login', (req, res) => {
```
Comment (valentinbreiz) [inline routes/user.js:17]: Why re add old files? We moved to GRPC.

File: package.json [hunk 3]
```diff
@@ -16,5 +16,10 @@
     "firebase": "^10.0.0",
     "google-protobuf": "^3.21.2",
     "grpc-tools": "^1.12.4"
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
```
Comment (valentinbreiz) [inline package.json:20]: Express should be removed since we use GRPC.

**Review by valentinbreiz — COMMENTED** [review]
Please remove REST routes + push your protos in the Honeypot repository (here https://github.com/Honeybrain/Honeypot/blob/main/protos/user.proto)

**Pull request closed without merging**